giza_actions/integrations/uniswap/pool.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from ape import Contract, chain
+
+from .utils import tick_to_price
+
+
+class Pool:
+    def __init__(self, address, sender, token0=None, token1=None, fee=None):
+        self.contract = Contract(
+            address, abi=os.path.join(os.path.dirname(__file__), "ASSETS/pool.json")
+        )
+        self.sender = sender
+        self.token0 = Contract(
```

Comment:
Are all of these variables supposed to be used by the user? We need to think what we tell the user "This is for you to use" or "This is 'private' and use at your own risk"